docker/Dockerfile

```diff
@@ -1,79 +0,0 @@
-FROM php:7.1-apache
```

Comment:
Should there be something in the Release Notes about the rename of the Docker files?

Reply:
These are only for local development, CI and demo purposes - @echarrod is it worth mentioning this?

Reply:
I think we could add it to our internal dev process wiki, but no worth adding to the README if clients/public don't need to know about them